docs/software/container-development-guide.md

```diff
@@ -8,20 +8,24 @@ This document contains instructions for OSG Technology Team members, including:
 - How to manage tags for images in the [OSG DockerHub organization](https://hub.docker.com/r/opensciencegrid/)
 - Tips for container image development
 
-Creating a New Container Image
-------------------------------
 
-### Prepare the GitHub repository ###
+New Containers Using GitHub Actions
+-----------------------------------
 
-1. Create a Git repository whose name is prefixed with `docker-`, e.g. `docker-frontier-squid`
+### Creating a new container image (GitHub Actions) ###
```

Comment:
```suggestion
### Creating a new container image ###
```

Section headers with the same name in a doc are numbered so we can have duplicates

Reply:
How do we disambiguate between them in links?

Reply:
I forget the exact formatting but this section would look something like `#creating-a-new-container-image1` and the Travis CI subsection could be referred to with `#creating-a-new-container-image2`.